CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+All notable changes to this project will be documented in this file.
```

Comment:
Des-Quorum is a fork of JPMorgen Quorum so all notable changes to this fork will be documented in this file.